sfa_dash/templates/data/table/site_table.html

```diff
@@ -20,7 +20,12 @@
     <tr>
       <td class="name-column"><a href="{{ row['link'] }}">{{ row['name'] }}</a></td>
       <td class="provider-column">{{ row['provider'] }}</td>
-      <td class="climate-zones-column">{{ row['climate_zones'] | join(', ') }}</td>
+      {% if row['climate_zones'] |length > 0 %}
+      {% set climate_zones = row['climate_zones'] |join(', ') %}
+      {% else %}
+      {% set climate_zones = "No Climate Zone" %}
```

Comment:
Should we define this at a higher level so that it can be referenced on, say, a site page? I don't know where to suggest. I also wouldn't list this like a normal zone on a site page, so maybe just ignore me and leave as is for now.

Reply:
Updates to this move the linking logic to a filter so that it can also be used on site pages, but only print the zones if the list isn't empty, so "No Climate Zones" is not displayed.